src/twisted/internet/defer.py

```diff
@@ -53,6 +53,17 @@ class TimeoutError(Exception):
 
 
 
+class NoCurrentResult(BaseException):
+    """
+    This error is raised when attempting to call L{extractCurrentResult} on a
```

Comment:
Looks like now this is raised by both functions. Is mentioning one better than either mentioning both or neither and just describing the situation (I have no opinion, so maybe it's fine as is)